src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -134,6 +134,8 @@ As a convenient shorthand, `transition:persist` can alternatively take a transit
 <video controls="" autoplay="" transition:persist="media-player">
 ```
 
+When you add `transition:persist` to an island, the state is retained upon navigation, but any props that change will be carried over to the new page, causing your component to re-render with new props, but the existing state. You can control this behavior with the `transition:persist-props` directive. Adding this directive will keep the existing props and not copy over new values.
```

Comment:
```suggestion
#### `transition:persist-props`
<p><Since v="4.5.0" /></p>
**Default**: `false`

This allows you to control whether or not an island's props should be persisted upon navigation.

By default, when you add `transition:persist` to an island, the state is retained upon navigation, but your component will re-render with new props. This is useful, for example, when a component receives page-specific props such as the current page's `title`.

You can override this behavior with `transition:persist-props="true"`. Adding this directive will keep an island's existing props (not re-render with new values) in addition to maintaining its existing state.
```
Here's my suggestion, so we can mention when this was introduced too.

Reply:
You don't need the `="true"` part but otherwise very good! I can't edit the suggestion so I think i'll commit and then change.